src/PQueue.js

```diff
@@ -0,0 +1,48 @@
+'use strict'
+
+export default class PQueue {
+
+  constructor() {
+    this._list = []
```

Comment:
I'm personally not a fan of the `_` naming convention - for me, it doesn't add much to the meaning of the identifier (though I happen to know that this used to be a convention for private variables in C), so I'm guessing that other folks may not benefit from this practice.  Just a personal style note.  😄